CircleandCross2/Program.cs

```diff
@@ -20,32 +20,86 @@ static void GameBoard (string[,] board)
 			
 		}
 
+		static bool GameWonCondition(string[,] board)
+		{
+			for (int i = 0; i < 3; i++)
+			{
+				for (int j = 0; j < 3; j++)
+				{
+					string arr = board[i, j];
+
+					if (board[0, 0] == "[o]" && board[0, 1] == "[o]" && board[0, 2] == "[o]"
+					|| board[1, 0] == "[o]" && board[1, 1] == "[o]" && board[1, 2] == "[o]"
+					|| board[2, 0] == "[o]" && board[2, 1] == "[o]" && board[2, 2] == "[o]"
+					|| board[0, 0] == "[o]" && board[1, 1] == "[o]" && board[2, 2] == "[o]"
+					|| board[0, 0] == "[o]" && board[1, 0] == "[o]" && board[2, 0] == "[o]"
+					|| board[0, 1] == "[o]" && board[1, 1] == "[o]" && board[2, 1] == "[o]"
+					|| board[0, 2] == "[o]" && board[1, 2] == "[o]" && board[2, 2] == "[o]"
+					|| board[0, 2] == "[o]" && board[1, 1] == "[o]" && board[2, 0] == "[o]"
+					|| board[0, 0] == "[x]" && board[0, 1] == "[x]" && board[0, 2] == "[x]"
+					|| board[1, 0] == "[x]" && board[1, 1] == "[x]" && board[1, 2] == "[x]"
+					|| board[2, 0] == "[x]" && board[2, 1] == "[x]" && board[2, 2] == "[x]"
+					|| board[0, 0] == "[x]" && board[1, 1] == "[x]" && board[2, 2] == "[x]"
+					|| board[0, 0] == "[x]" && board[1, 0] == "[x]" && board[2, 0] == "[x]"
+					|| board[0, 1] == "[x]" && board[1, 1] == "[x]" && board[2, 1] == "[x]"
+					|| board[0, 2] == "[x]" && board[1, 2] == "[x]" && board[2, 2] == "[x]"
```

Comment:
Make the string with the player filed type as a parameter of the function:
`static bool GameWonCondition(string[,] board, string playerField)` and use it to compare with the values in the board game array.